eb_sqs/auto_tasks/service.py

```diff
@@ -54,12 +55,12 @@ def _auto_task_wrapper(module_name, class_name, func_name, *args, **kwargs):
                 exc.max_retries_func()
             else:
                 # by default log an error
-                logger.error('Reached max retries in auto task {}.{}.{} with error: {}'.format(module_name, class_name, func_name, repr(exc)))
+                logger.error('Reached max retries in auto task {}.{}.{} with error: {}'.format(module_name, class_name,
```

Comment:
Should we convert these to f-string way now?

Reply:
There are other places that we use the old .format way, but not sure if it is too much work or not to convert all to f-string. What do you think?

Reply:
i think we can just leave it as is. there's too many lines of code that use the old style and it's not worth the effort to convert imo - unless Python plans to not support that old style format anytime soon.